@hubblecommerce/hubble/core/components/swComponents/slots/manufacturer-logo-slot.vue

```diff
@@ -44,7 +44,8 @@ export default {
             return this.content && this.content.data && this.content.data.media;
         },
         linkUrl() {
-            return this.content && this.content.data && this.content.data.url;
+            return (this.content && this.content.data && this.content.data.url) ||
```

Comment:
Content.data.url still needed? Why checking both?

Reply:
If element is placed on CMS- / Landingpage the URL is in `content.data.url`.
Otherwise - on product page - its empty and found in `content.data.manufacturer.link`.